mscore/palette.cpp

```diff
@@ -1045,6 +1158,15 @@ bool Palette::event(QEvent* ev)
                qApp->translate("Palette", cellAt(idx)->name.toUtf8()), this);
             return false;
             }
+      else if (ev->type() == QEvent::KeyPress) {
```

Comment:
I still don't get why this is needed. QWidget::event() would the same (calling `keyPressEvent`) but it will return the result of `keyPressEvent`. Here you always return true and so stop the propagation of the event to the parent. Why is that needed again? Add a comment.